### PR TITLE
adding ESLint

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true
```
Comment (baksofill) [inline .eslintrc.js]: do we need es6 now? 
Maybe you thought about es5?

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 2015
```
Comment (baksofill) [inline .eslintrc.js]: do we need es6?

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ]
+    }
+};
```
Comment (baksofill) [inline .eslintrc.js]: please add empty line at the end

Comment (dshovchko) [reply .eslintrc.js]: I have added eol-last rule to eslintrc file and removed es6 option

**Review by baksofill — APPROVED** [review]

**Pull request merged**